### ISSUE TITLE
hip cmake exports error

### ISSUE THREAD

**Issue by cyrush** [Alpine-DAV/ascent#979]
exporting BLT hip targets and and upon import calling `find_package(hip)`, still have an issue with missing AMD cmake targets:

as seen in using-with-cmake example when HIP is on:

```
CMake Error at /cmake-3.23.2-linux-x86_64/share/cmake-3.23/Modules/CMakeFindDependencyMacro.cmake:47 (find_package):
  By not providing "FindAMDDeviceLibs.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "AMDDeviceLibs", but CMake did not find one.

  Could not find a package configuration file provided by "AMDDeviceLibs"
  with any of the following names:

    AMDDeviceLibsConfig.cmake
    amddevicelibs-config.cmake

  Add the installation prefix of "AMDDeviceLibs" to CMAKE_PREFIX_PATH or set
  "AMDDeviceLibs_DIR" to a directory containing one of the above files.  If
  "AMDDeviceLibs" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  /opt/rocm/lib/cmake/hip/hip-config.cmake:187 (find_dependency)
  /home/user/ascent/install/lib/cmake/ascent/ascent_setup_deps.cmake:122 (find_package)
  /home/user/ascent/install/lib/cmake/ascent/AscentConfig.cmake:129 (include)
  CMakeLists.txt:38 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/user/ascent/install/examples/ascent/using-with-cmake/build/CMakeFiles/CMakeOutput.log".
See also "/home/user/ascent/install/examples/ascent/using-with-cmake/build/CMakeFiles/CMakeError.log".
+ set -e
+ /cmake-3.23.2-linux-x86_64/bin/cmake --build build
make: Makefile: No such file or directory
make: *** No rule to make target 'Makefile'.  Stop.
```

**Comment by cyrush** [comment]
Found the issue: Hip CMake logic requires `ROCM_PATH` to be set (you can't just call `find_package` and give it a path, the CMake var ROCM_PATH must be set)

**Comment by cyrush** [comment]
resolved in #987
